DataEng: Add test tasks and docker-compose for local deployment (#25)
provectus/internship: dataeng/README.md

> ### Prerequisite

Reviewer: ```suggestion
### Prerequisites
```

* Python3.7 or greater
* Docker 19.03 or greater
* Git 2.28 or greater
* Postgres 13 or greater

## Level 1

### Files definitions:

- src_data - Path with source data needed to be processed.
- processed_data - Path with output processed data.
- user_id.jpg - User image file, for example, 0001.jpg. Could be several for different users in source data path.
- user_id.csv - User info file, for example, 0001.csv. Could be several for different users in source data path.

User csv file contains next columns:

1. first_name - User first name
2. last_name - User last name
3. birthts - User birthdate timestamp in milliseconds UTC

**For example:**

```text
first_name, last_name, birthts
Ivan, Ivanov, 946674000000
```

### Data processing description

> 1. Read csv file

Reviewer: А где находится этот csv файл?

> 2. Try to find image for the same user.

Reviewer: Может
_Match images for each user_

3. Combine data from CSV and image path

> 4. Update processed_data/output.csv CSV file and add new data. Important we can update data for previously processed

Reviewer: _Combine the csv file data with image paths_?

>    user. In output CSV and DB we should not to duplicate records. Output CSV file format: user_id, first_name,

Reviewer: ```suggestion
   user. In output CSV and DB we should not duplicate records. Output CSV file format: user_id, first_name,
```

   last_name, birthts, img_path

## Task

> Implement script to process files in src_data folder.

Reviewer: ```suggestion
Implement a script to process files from the `src_data` folder.
```

## Results delivery format

Results should be implemented as a python script with demo data. Also should be
provided the README.md file with the description of your solution.

## Level 2

> The same as **Level 1** but with next differences.

Reviewer: ```suggestion
The same as **Level 1** with the following extras.
```

## Results delivery format

> Results should be implemented as a service. The service should periodically read source data and process it.

Reviewer: Имеется в виду **web service**?
